common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -598,11 +588,10 @@ bool mon_wlan_hal_dwpal::update_radio_stats(SRadioStats &radio_stats)
         }
     }
 
-    calc_curr_traffic((uint64_t)BytesSent, radio_stats.tx_bytes_cnt, radio_stats.tx_bytes);
-    calc_curr_traffic((uint64_t)BytesReceived, radio_stats.rx_bytes_cnt, radio_stats.rx_bytes);
-    calc_curr_traffic((uint64_t)PacketsSent, radio_stats.tx_packets_cnt, radio_stats.tx_packets);
-    calc_curr_traffic((uint64_t)PacketsReceived, radio_stats.rx_packets_cnt,
-                      radio_stats.rx_packets);
+    calc_curr_traffic(BytesSent, radio_stats.tx_bytes_cnt, radio_stats.tx_bytes);
```

Comment:
(nitpick)  
Not your code, but since you are here, we use lower_snake_case for local variables.